src/Contexts/NetworkManagerContext.tsx

```diff
@@ -136,14 +136,14 @@ const NetworkManagerProvider = ({ children }: INetworkManagerProviderProps) => {
         setDestinationChains(
           chainbridgeConfig.chains.filter(
             (bridgeConfig: BridgeConfig) =>
-              bridgeConfig.destinationChain === true
+              bridgeConfig.chainId !== chain.chainId
```

Comment:
How will it work for user now?

Reply:
As we don't have the same chain Id for the two blockchains, it will work as expected. The blockchain which has been set as home chain wont be shown under the destination chain.